docs/fundamentals/setup-tooling.md

```diff
@@ -136,7 +139,16 @@ For more information, see [.NET Aspire templates](aspire-sdk-templates.md).
 
 ## Container runtime
 
-.NET Aspire projects are designed to run in containers. You can use either Docker Desktop or Podman as your container runtime. [Docker Desktop](https://www.docker.com/products/docker-desktop/) is the most common container runtime. [Podman](https://podman.io/docs/installation) is an open-source daemonless alternative to Docker, that can build and run Open Container Initiative (OCI) containers. If your host environment has both Docker and Podman installed, .NET Aspire defaults to using Docker. You can instruct .NET Aspire to use Podman instead, by setting the `ASPIRE_CONTAINER_RUNTIME` environment variable to `podman`:
+.NET Aspire supports running containers using various OCI-compatible runtimes, such as Docker Desktop, Podman, and Rancher Desktop.
```

Comment:
Technically, what we really require is not only an OCI-compatible runtime, but also a CLI interface to that runtime. And we currently support only two flavors of the CLI, that is `docker` CLI and `podman` CLI. That is why Rancher Desktop works in "Docker mode" but it does not in "nerdctl mode". 

Do you think this nuance is important enought to be included in the docs?

Reply:
> Do you think this nuance is important enough to be included in the docs?

Yes, absolutely.